### ISSUE TITLE
How do I deal with this error?

### ISSUE THREAD

**Issue by shivakrishna9** [ArunMichaelDsouza/tensorflow-image-detection#1]
![outputs](https://user-images.githubusercontent.com/15874356/26932951-c2021276-4c82-11e7-8a9b-947fb88f7099.png)
This is the screenshot of the output, when I ran the train.sh file

**Comment by ArunMichaelDsouza** [comment]
Seems like the label for the second training set was not generated, you sure you provided the training dataset in the specified manner ?

**Comment by shivakrishna9** [comment]
Yes. I provided it.
Also Will there be a problem if all the images are not of same size?

**Comment by ArunMichaelDsouza** [comment]
No it does not matter, what matters is the no of images present in a particular dataset (at least 100 for each dataset is recommended). Also, the more different angles covered (of the subject), the better the training session will be, and will result in a better prediction ultimately. As long as you've provided the data sets in the specified format, there shouldn't be a problem.

**Comment by shivakrishna9** [comment]
Thank you. It worked with flowers data set from tensorflow tutorial. How did you create the image output file consisting images and prediction scores?

**Comment by shivakrishna9** [comment]
https://stackoverflow.com/questions/38175673/critical-tensorflowcategory-has-no-images-validation. 
Issue has resolved. I am closing the issue.

**Comment by shivakrishna9** [comment]
https://stackoverflow.com/questions/38175673/critical-tensorflowcategory-has-no-images-validation. 
Issue has resolved. I am closing the issue.
